src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -290,6 +307,41 @@ namespace
             {
                 assert(next_mode != ScatteringMode::Absorption);
 
+                //Count the number of diffuse bounces
+                if (ScatteringMode::has_diffuse(next_mode))
+                {
```

Comment:
For convenience and improved code density, you can omit curly braces for single-line blocks:

    if (ScatteringMode::has_diffuse(next_mode))
        ++m_diffuse_bounces;